libs/@hashintel/type-editor/src/entity-type-editor/link-list-card.tsx

```diff
@@ -181,7 +183,7 @@ const LinkTypeRow = ({
           </EntityTypeTableTitleCellText>
         </TableCell>
         <TableCell sx={{ py: "0 !important" }}>
-          <LinkEntityTypeSelector linkIndex={linkIndex} />
+          <DestinationEntityTypeSelector linkIndex={linkIndex} />
```

Comment:
Renamed a few components for clarity – there are two 'link entity type selectors' in the link table, the one where you add a new link to the entity, and the one where you add a new link as a destination for a link type. this is the latter

Reply:
Honestly I think this is more immediately understandable than 'right entity type', and until we introduce bidirectional links it's accurate